### PR TITLE
Fix RTL Styles

### LINKED ISSUE #12450: EQM - Replace questions in RTL language - The checkboxes and arrows are overlapping
## Observed behavior

See the video. The checkboxes and arrows are overlapping but it's still possible to select them.

Video:

https://github.com/user-attachments/assets/55c0a90f-cb8e-4a1f-86b9-750a54029a78

## Expected behavior

The checkboxes and arrows should not be overlapping.

## Steps to reproduce the issue

1. Install the build from https://github.com/learningequality/kolibri/releases/tag/v0.17.0-beta3
2. Create a new quiz with sections.
3. Attempt to replace questions while viewing in Arabic.

## Usage Details
Ubuntu 22  - Chrome

### PR DESCRIPTION
**Pull request by AlexVelezLl**
## Summary
Fix RTL Styles

| Before | After |
| --- | --- |
| Accordion question item in index view |  |
| image | image |
| Accordion question item in replace questions |  |
| image | image |
| Bottom navigation in replace questions |  |
| image | image |
| Bottom navigation in select resource |  |
| image | image |
| Bottom navigation in edit section |  |
| image | image |
| Edit section order |  |
| image | image |
| More button on tabs overflow |  |
| image | image |

## References

Closes #12450

## Reviewer guidance
Play with quiz creation using a RTL language, and check that styles are compliant with RTL.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue [hunk 1]
```diff
@@ -1006,15 +993,14 @@
     right: 0;
     bottom: 0;
     left: 0;
+    display: flex;
+    justify-content: space-between;
```
Comment (nucleogenesis) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue:997]: I dig this move to using flex + space-between justification to spread things evenly. I wonder how many uses of KGrid we have that could be far more easily solved with a couple lines of CSS.

**Review by nucleogenesis — APPROVED** [review]
Overall the code changes LGTM - I tested things out locally and the changes look good to me both in code and on screen. Thanks Alex!

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue (hunk 1 above)
Comment (AlexVelezLl) [reply kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue:997]: Yeees! I love flex for these more specific layouts.

**Comment by radinamatic** [comment]
Admittedly I am not @pcenov , and this is a type of PR where I miss his eyes the most, but I can confirm that the above alignment issues seem fixed 🙂 

RTL


Here I'm just nitpicking, so feel free to open as a follow up issue with low priority, but the `+Add section` button is a little jumpy and does not look left-aligned all the time.

RTL

**Comment by AlexVelezLl** [comment]
Thank you @radinamatic! I think the add section button issue is likely to be fixed in #12449, if it isn't, I can open a follow up issue for this :).

**Pull request merged**